### ISSUE TITLE
add JSONSync detailed usage to docs

### ISSUE THREAD

**Issue by linglejack06** [team401/Docs#28]
- description of what to use for
- how to make constants loader class
- how to use json exclude for special types

**Comment by avidraccoon** [comment]
Can this wait a little because after seeing the problems with when you were using it on high key I was gonna make major changes to improve it's usage because I have now seen many shortsighted decisions I had made.

**Comment by avidraccoon** [comment]
@linglejack06

**Comment by linglejack06** [comment]
Yeah, I can wait a bit. Although it seems designed pretty well to me other than a few readability quirks, what issues did you find with it?Sent from my iPhoneOn Nov 22, 2024, at 5:51 PM, avidraccoon ***@***.***> wrote:﻿
@linglejack06

**Comment by avidraccoon** [comment]
Some unimplemented features

**Comment by avidraccoon** [comment]
@linglejack06 Did you still want to do this or do you want me to do it?

**Comment by linglejack06** [comment]
you can do it

**Comment by avidraccoon** [comment]
@linglejack06 How do I write the docs?

**Comment by avidraccoon** [comment]
This will need to be delayed because I am currently working on changes to JSONSync
